DFC.Digital/DFC.Digital.Web.Sitefinity.CourseModule.UnitTests/Views/CourseDetailsViewTests.cs

```diff
@@ -304,6 +305,45 @@ public void ShowOtherVenuesAndDatesTest(bool otherVenuesExists)
             }
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void OtherVenuesStartdatesTest(bool startDateIsNull)
+        {
+            //Arrange
+            var startDate = DateTime.Now;
+
+            var courseDetailsCourseDetails = new _MVC_Views_CourseDetails_OtherDatesAndVenues_cshtml();
+            var courseDetailsViewModel = new CourseDetailsViewModel();
+            var courseDetails = new CourseDetails();
+
+            courseDetailsViewModel.CourseDetails.VenueDetails = new Venue();
+            courseDetailsViewModel.CourseDetails.Oppurtunities = new System.Collections.Generic.List<Oppurtunity>();
+            var otherDatesAndVenues = new Oppurtunity();
+            if (!startDateIsNull)
+            {
+                otherDatesAndVenues.StartDate = startDate;
+            }
```

Comment:
If "new Oppurtunuty()" now sets "StartDate = DateTime.Now"... isn't "otherDatesAndVenues.StartDate" always going to be be set to DateTime.Now irrespective of whether "if (!startDateIsNull)" is true or not?